suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -346,11 +346,16 @@ const authConfirm = (draft: State, device: TrezorDevice, success: boolean) => {
 const createInstance = (draft: State, device: TrezorDevice) => {
     // only acquired devices
     if (!device || !device.features) return;
+
+    const isPortfolioTrackerDevice = device.id === PORTFOLIO_TRACKER_DEVICE_ID;
+
     const newDevice: TrezorDevice = {
         ...device,
         passphraseOnDevice: false,
         remember: false,
-        state: undefined,
+        // In mobile app, we need to keep device state defined by the constant
+        // to be able to filter device accounts for portfolio tracker
+        state: isPortfolioTrackerDevice ? device.state : undefined,
```

Comment:
Can you explain little bit more? I probably don't understand why we can't have device with state.

Reply:
We need the device state, but when we call createImportedDeviceThunk, that triggers this reducer action and  the device state that we defined inside the constant gets overriden. That's good for devices from connect as they handle their own state.

Reply:
Thanks, makes sense.